### ISSUE TITLE
Mesh not showing up inworld

### ISSUE THREAD

**Issue by fly-man-** [WhiteCoreSim/WhiteCore-Dev#298]
### Steps to reproduce

1. Load random OAR file with Mesh build
2.  Restart region

### Expected behaviour

The Mesh should appear rezzed

### Actual behaviour

The Mesh isn't showing up

### Server configuration

**Operating system**:

Ubuntu 16.10 / x64

**Web server:**

Nginx

**Database:**

MariaDB

**WhiteCore version:**

Master

### Logs

Binary LLSD parsing: Unknown type marker: 110
Apr 04 03:23:12 - Failed to decode mesh asset
Apr 04 03:23:12 - System.NullReferenceException: Object reference not set to an instance of an object
  at OpenMetaverse.Assets.AssetMesh.Decode () [0x00040] in <dc9dede8d1c843ffa1fbda11de39e640>:0

**Comment by emperorstarfinder** [comment]
I am sue you already did this, but for comedy sake have you tried to reproduce with multiple OARs that have mesh in them (different OARS of course) just to be sure it is not that particular OAR?

**Comment by fly-man-** [comment]
@emperorstarfinder Yes, for some reason it just stops rendering these mesh items. I've tried it with multiple OAR files, even created a clean one just to check.

I am not deep into Mesh to know what this marker 110 exactly stands for, perhaps that might be also part of the solution

**Comment by emperorstarfinder** [comment]
can you email me the oars and ill test them if possible?  I might be able to figure out whats going wrong  If it is an OpenSim created OAR then that might be a problem we may not have a marker they do as well.

**Comment by fly-man-** [comment]
I'll grab the OAR from my development machine when I am back home.

**Comment by greythane** [comment]
This looks like a problem in the OMV libraries.  
I suspect there may be newer definitions added that we do not have yet.
I'll try and check this out.

**Comment by BogusCurry** [comment]
I will test it on weekend on my local grid ;D

**Comment by emperorstarfinder** [comment]
This is why I am having him email me the OARs so I can see what happens when the OAR loads.  the 110 marker is a marker we should already have which is why I want to see what happens when I try to load the oars.

**Comment by andsim** [comment]
is this fix yet?
